var/spack/repos/builtin/packages/fmt/package.py

```diff
@@ -14,6 +14,8 @@ class Fmt(CMakePackage):
     homepage = "https://fmt.dev/"
     url      = "https://github.com/fmtlib/fmt/releases/download/7.1.3/fmt-7.1.3.zip"
 
+    version('8.1.1', sha256='23778bad8edba12d76e4075da06db591f3b0e3c6c04928ced4a7282ca3400e5d')
```

Comment:
Also from upstream?

Reply:
Yes, I just reverted 2022 -> 2021 in licences because...